examples/euler_vanderpol.py

```diff
@@ -86,7 +85,7 @@ def jac(t,y):
 
     #Basic test
     x1 = y[:,0]
-    nose.tools.assert_less(np.abs(float(x1[-1]) - 1.8601438), 1e-1)
+    assert np.abs(float(x1[-1]) - 1.8601438) < 1e-1
```

Comment:
Just a minor comment that we are a bit inconsistent with `np.abs` vs `abs` (scalars in both cases)